examples/tutorials/VHACD_tutorial.py

```diff
@@ -0,0 +1,234 @@
+# [setup]
+import math
+import os
+import time
+
+import magnum as mn
+import numpy as np
+
+import habitat_sim
+from habitat_sim.utils import viz_utils as vut
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+data_path = os.path.join(dir_path, "../../data")
+output_path = os.path.join(dir_path, "VHACD_tutorial_output/")
+
+if not os.path.exists(output_path):
+    os.mkdir(output_path)
+
+# Define Helper Functions
+def remove_all_objects(sim):
+    for id_ in sim.get_existing_object_ids():
+        sim.remove_object(id_)
+
+
+def make_configuration():
+    # simulator configuration
+    backend_cfg = habitat_sim.SimulatorConfiguration()
+    backend_cfg.scene_id = "NONE"
+
+    backend_cfg.enable_physics = True
+
+    # sensor configurations
+    sensor_specs = []
+    sensor_spec = habitat_sim.SensorSpec()
+    sensor_spec.uuid = "rgba_camera_1stperson"
+    sensor_spec.sensor_type = habitat_sim.SensorType.COLOR
+    sensor_spec.resolution = [544, 720]
+    sensor_specs.append(sensor_spec)
+
+    # agent configuration
+    agent_cfg = habitat_sim.agent.AgentConfiguration()
+    agent_cfg.sensor_specifications = sensor_specs
+
+    return habitat_sim.Configuration(backend_cfg, [agent_cfg])
+
+
+def simulate(sim, dt=1.0, get_frames=True, data=None):
```

Comment:
If we're not using the metric collection/logging for the tutorial script let's remove it.